conjur/argument_parser/_screen_options_parser.py

```diff
@@ -0,0 +1,25 @@
+from conjur.version import __version__
+from conjur.argument_parser.parser_utils import conjur_copyright
+
+
+class ScreenOptionsParser:
```

Comment:
please add a docstring here something like

"""
Subclass for the main options landing screen and is part of the bigger ArgparserBuilder class. This subclass was separated from the bigger class to better organize the code and increase readability
"""

Please add this docstring replacing 'main options landing screen' of course with each subclass's responsibility